tests/system/test_mixins.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pytest
+
+from google.api_core import exceptions
+
+
+def test_get_operation(echo):
+    with pytest.raises(exceptions.NotFound):
+        echo.get_operation({"name": "operations/**"})
+
+
+def test_list_operations(echo):
+    response = echo.list_operations({"name": "operations/name"})
+    assert response.operations[0].name == "a/pending/thing"
+
+
+def test_delete_operation(echo):
+    echo.delete_operation({"name": "operations/name"})
+
+
+def test_cancel_operation(echo):
+    echo.cancel_operation({"name": "operations/name"})
+
```

Comment:
@parthea what are these two tests doing? Should we add an assert in them?

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/2175/commits/ff126262eb70936290abc9f1ef6ce345b9cc8d5f